src/System.Private.CoreLib/src/System/Resources/FileBasedResourceGroveler.cs

```diff
@@ -93,17 +92,22 @@ private String FindResourceFile(CultureInfo culture, String fileName)
             // look in .
             if (File.Exists(fileName))
                 return fileName;
-
+                
             return null;  // give up.
         }
 
-        // Constructs a new ResourceSet for a given file name.
+        // Constructs a new ResourceSet for a given file name.  The logic in
+        // here avoids a ReflectionPermission check for our RuntimeResourceSet
+        // for perf and working set reasons.
         private ResourceSet CreateResourceSet(String file)
         {
             Debug.Assert(file != null, "file shouldn't be null; check caller");
 
             if (_mediator.UserResourceSet == null)
             {
+                // Explicitly avoid CreateInstance if possible, because it
+                // requires ReflectionPermission to call private & protected
```

Comment:
These comments about ReflectionPermission  refer to code-access security that is gone in .NET Core. They should be deleted instead.

Reply:
OK, I'll remove from corert as well